src/tools/extract.cpp

```diff
@@ -46,6 +46,11 @@ return_code parseArguments(int argc, char *argv[], extractor::ExtractorConfig &e
             ->implicit_value(true)
             ->default_value(false),
         "Generate a lookup table for internal edge-expanded-edge IDs to OSM node pairs")(
+        "dump-partition-graph",
+        boost::program_options::value<bool>(&extractor_config.dump_compressed_node_based_graph)
+            ->implicit_value(true)
+            ->default_value(false),
+        "Generate a partitionable graph file (.cnbg) for use with osrm-partition")(
```

Comment:
Hm on the demo server with Boost 1.55 I _have to_ say `--dump-partition-graph true` even though `implicit_value` above should only require `dump-partition-graph`. No idea if this is an Boost issue...

http://www.boost.org/doc/libs/1_55_0/doc/html/boost/program_options/typed_value.html#idp101668472-bb